homework/jenya_s/homework7.py

```diff
@@ -0,0 +1,12 @@
+def magic_range():
```

Comment:
magic range shouldn't be generator. This function has to return formatted string: `[-5; 257)`.